docs/products/redis/concepts/warning-overcommit_memory.rst

```diff
@@ -0,0 +1,8 @@
+Warning ``overcommit_memory``
+=============================
+
+By starting a Redis service on `Aiven console <https://console.aiven.io/>`_, you may notice on **Logs** the following **warning** ``overcommit_memory``::
```

Comment:
```suggestion
When starting a Redis service on `Aiven console <https://console.aiven.io/>`_, you may notice on **Logs** the following **warning** ``overcommit_memory``::
```

Is it when starting the service with the Console UI or with the terminal? I suspect it doesn't matter how you start the service, in this case I would remove " on `Aiven console <https://console.aiven.io/>`_"